README.md

```diff
@@ -20,10 +22,16 @@ optional):
   * tags   - A list of tags describing this package.
   * description - A description of this package.
   * license - The license of the source code in the package.
-  * web    - An optional url for humans to read additional information about
+  * web    - An optional URL for humans to read additional information about
              the package.
+  * doc    - An optional URL for humans to read the package HTML documentation
+
+### Requirements
+
+While we really appreciate your contribution, please ensure that you package matches the following requirements: other developers rely on your package. Non-compliant packages might be removed with no warning.
```

Comment:
> following requirements: other developers rely on your package

That seems like a catch-22. A package can't get to the package list because no developers rely on it, no developers rely on it because it's not in the package list...